apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,3 +157,11 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
+
+.conversation-view .panel {
```

Comment:
Haven't found smarter way that will work for both hybrid(separated inbox, but multi-functional conversation)/monolithic views yet :) Maybe you have better idea?